lib/brakeman/checks/check_reverse_tabnabbing.rb

```diff
@@ -0,0 +1,43 @@
+require 'brakeman/checks/base_check'
+
+class Brakeman::CheckReverseTabnabbing < Brakeman::BaseCheck
+  Brakeman::Checks.add_optional self
+
+  @description = "Checks for reverse tabnabbing cases on 'link_to' calls"
+
+  def run_check
+    calls = tracker.find_call :methods => :link_to
+    calls.each do |call|
+      process_result call
+    end
+  end
+
+  def process_result result
+    return unless original? result and result[:call].third_arg
+
+    html_opts = result[:call].third_arg
+    _, target = hash_access html_opts, :target
```

Comment:
`hash_access` does not check if the value is actually a Hash node. It should be checked with `hash?` before calling `hash_access`.

There is no guarantee the value will be a string Sexp. The value should be checked with `string?` and then you can get the string value with `target.value`. Same for `rel` below.